### PR TITLE
[FIX] Add multiple instances of the same server

### LINKED ISSUE #669: [BUG] You can add multiple instances of the same server
We shouldn't be able to add multiple instances of the same server.
The expected behavior is to just redirect the user to the currently existing server.

|Image|
|-|
||

### PR DESCRIPTION
**Pull request by macabeus**
@RocketChat/ios

Closes #669

### CODE REVIEW

**Comment by rafaelks** [comment]
@macabeus Thank you for your PR! You need to sign the CLA so we can merge the PR.

**Comment by macabeus** [comment]
@rafaelks Ok, I made it now =]

File: Rocket.Chat/Controllers/Auth/ConnectServerViewController.swift [hunk 1]
```diff
@@ -125,6 +125,23 @@ final class ConnectServerViewController: BaseViewController {
...
+                    let serverUrl = URL(string: stringServerUrl) {
+
+                    return serverUrl == socketURL
+                } else {
+                    return false
+                }
+            })
+
+            if let sameServerIndex = sameServerIndex {
+                MainChatViewController.shared?.changeSelectedServer(index: sameServerIndex)
```
Comment (rafaelks) [inline Rocket.Chat/Controllers/Auth/ConnectServerViewController.swift]: What if `MainChatViewController.shared` doesn't exists here?

Comment (macabeus) [reply Rocket.Chat/Controllers/Auth/ConnectServerViewController.swift]: Sorry, but I don't know... What do you prefer that I do?

**Pull request merged**